### ISSUE TITLE
Cannot build 1.1.0

### ISSUE THREAD

**Issue by protist** [ibancg/lingot#50]
I'm struggling to build the latest stable version 1.1.0. I can build the previous version 1.0.1 perfectly. I'm on an up-to-date Arch Linux with `perl-xml-parser` and `intltool` installed.
```
mv -f src/.deps/lingot-lingot-io-config-scale.Tpo src/.deps/lingot-lingot-io-config-scale.Po
gcc -DHAVE_CONFIG_H -I.    -pthread -Wall -Wextra -pedantic -Wno-overlength-strings -I./src     -D_REENTRANT  -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/gio-unix-2.0 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/at-spi-2.0 -pthread  -I/usr/include/json-c  -g -O2 -MT src/lingot-lingot-io-config.o -MD -MP -MF src/.deps/lingot-lingot-io-config.Tpo -c -o src/lingot-lingot-io-config.o `test -f 'src/lingot-io-config.c' || echo './'`src/lingot-io-config.c
src/lingot-io-config.c: In function ‘lingot_io_config_load_json’:
src/lingot-io-config.c:641:20: error: ‘TRUE’ undeclared (first use in this function)
  641 |     json_bool ok = TRUE;
      |                    ^~~~
src/lingot-io-config.c:641:20: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [Makefile:1435: src/lingot-lingot-io-config.o] Error 1
make[2]: Leaving directory '/HDD/protist/Downloads/lingot-1.1.0'
make[1]: *** [Makefile:1774: all-recursive] Error 1
make[1]: Leaving directory '/HDD/protist/Downloads/lingot-1.1.0'
make: *** [Makefile:943: all] Error 2
```

**Comment by ibancg** [comment]
Thanks for the report!, it should be fixed in latest commit 14ff1d2c872b304be04aabcfc7a51cdefed43293

**Comment by protist** [comment]
Thanks for the quick response @ibancg. I tested the commit, and it progressed further in the build, but it still wouldn't complete. I made a few more changes and #51 builds fine now.
